### PR TITLE
Improve content design on error pages

### PR DESCRIPTION
**Pull request by daniel-ac-martin**

### CODE REVIEW

File: html/partials/502.shtml [hunk 1]
```diff
@@ -1 +1,3 @@
-<p>An invalid response was received from the upstream service.</p>
+<h2>502 Bad gateway</h2>
+<p>We're having a technical problem.</p>
```
Comment (daniel-ac-martin) [inline html/partials/502.shtml]: "We currently have" ?

File: html/partials/503.shtml [hunk 2]
```diff
@@ -1,9 +1,10 @@
+<h2>503 Service unavailable</h2>
 <p>
-  Sorry, we are not currently able to fulfil your response. There are
-  two main situation in which this can occur:
+  Sorry, we can’t carry out your request at the moment. There are
+  two main situations in which this can occur:
```
Comment (daniel-ac-martin) [inline html/partials/503.shtml]: s/main/common/ ?

File: html/partials/504.shtml [hunk 3]
```diff
@@ -1 +1,3 @@
-<p>The upstream service failed to respond in a timely manner.</p>
+<h2>504 Gateway timeout</h2>
+<p>We’re having a technical problem.</p>
```
Comment (daniel-ac-martin) [inline html/partials/504.shtml]: "We currently have" ?

File: html/partials/50x.shtml [hunk 4]
```diff
@@ -1,2 +1,3 @@
-<p>Sorry, the page you are looking for is currently unavailable.</p>
-<p>Please try again later.</p>
+<h2>An error occurred</h2>
+<p>The page you’re looking for isn’t available.</p>
```
Comment (daniel-ac-martin) [inline html/partials/50x.shtml]: We should probably change this thinking about it. This page is really a catch-all.
It's difficult to know what to write.
Perhaps "We were unable to fulfil your request."?
@easternbloc: Any ideas?

File: html/partials/503.shtml [hunk 5]
```diff
@@ -1,9 +1,10 @@
...
 </p>
 <ol>
-  <li>Our service down for maintenance</li>
+  <li>Our service is down for maintenance</li>
```
Comment (daniel-ac-martin) [inline html/partials/503.shtml]: "Our service is being updated" ?
Also, should these lines start with a capital?
Should they terminate with a full-stop?

File: html/partials/50x.shtml (hunk 4 above)
Comment (daniel-ac-martin) [inline html/partials/50x.shtml]: Or we could go with "There's a problem with this service." like on the 500 page?..

File: html/partials/404.shtml [hunk 6]
```diff
@@ -1 +1,3 @@
-<p>Sorry, the page you are looking for was not found on this server.</p>
+<h2>404 Not found</h2>
+<p>We can’t find the page you’re looking for.</p>
+<p>Check that the web address is correct.</p>
```
Comment (daniel-ac-martin) [inline html/partials/404.shtml]: "Please check" ?

Comment (FolaEk) [inline html/partials/404.shtml]: We think - "If you entered a web address, please check it was correct."

File: html/partials/502.shtml (hunk 1 above)
Comment (FolaEk) [inline html/partials/502.shtml]: We - think:
"We're having a technical problem.
We’ll try and fix this as soon as possible."

File: html/partials/503.shtml (hunk 2 above)
Comment (FolaEk) [inline html/partials/503.shtml]: Sorry, we can't carry your request.

We think:

"Sorry, we can’t carry out your request at the moment. 
 
Please try again or come back later."

File: html/partials/504.shtml (hunk 3 above)
Comment (FolaEk) [inline html/partials/504.shtml]: We don't think we need to say "we currently have"

File: html/partials/50x.shtml (hunk 4 above)
Comment (FolaEk) [inline html/partials/50x.shtml]: We think: the content should be the same as on the 500 page so "There's a problem. Please try again in a few moments."

File: html/partials/500.shtml [hunk 7]
```diff
@@ -1,2 +1,3 @@
-<p><em>Something</em> went wrong.</p>
-<p>It's not you, it's us.</p>
+<h2>500 Internal server error</h2>
+<p>There's a problem with this service.</p>
+<p>Please try again in a few moments.</p>
```
Comment (FolaEk) [inline html/partials/500.shtml]: We think "There's a problem. Please try again in a few moments."

File: html/partials/503.shtml (hunk 5 above)
Comment (FolaEk) [inline html/partials/503.shtml]: We'd leave out the 2 scenarios, and just say we can't carry out the request.

File: html/partials/504.shtml (hunk 3 above)
Comment (daniel-ac-martin) [inline html/partials/504.shtml]: Then I shall leave it as it is.

**Pull request merged**